src/app/services/subject.ts

```diff
@@ -1,13 +1,5 @@
-import {TAG_ID} from "./";
-
 let id;
-if (["localhost:8000", "localhost:8004"].includes(document.location.host)  || document.location.host.includes("isaacphysics")) {
-    id = TAG_ID.physics;
-} else if (document.location.host == "localhost:8001" || document.location.host.includes("isaacchemistry")) {
-    id = TAG_ID.chemistry;
-} else if (document.location.host == "localhost:8002" || document.location.host.includes("isaacbiology")) {
-    id = TAG_ID.biology;
-} else if (document.location.host == "localhost:8003" || document.location.host.includes("isaaccomputerscience")) {
```

Comment:
I'm looking at this and wondering if you could just remove the whole concept of 'subject' in the class where this is consumed, or at least just statically return 'computer_science'. I'm not sure the 'unknown' case is going to matter, assuming that someone wants to access computer_science is a given

Reply:
Removed the entire file - seems like it isn't actually in use anywhere now.